CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Initial release of ferlab/postprocessing, created with the [nf-core](https://nf-
 ### `Added`
 
 ### `Fixed`
+[#1]https://github.com/FelixAntoineLeSieur/Post-processing-Pipeline/pull/1 Fixed template schemas
```

Comment:
I think that there are missing parenthesis around the url. Here is an example link taken from the markdown guide:
`My favorite search engine is [Duck Duck Go](https://duckduckgo.com)`

Reply:
oui je vais corriger ça